common.go

```diff
@@ -434,3 +441,20 @@ func GetLanguageByAlias(alias string) (lang string, ok bool) {
 
 	return
 }
+
+// GetLanguagesByGitAttributes returns the language assigned in .gitattributes if the regular expresion
+// matchs with the filename
```

Comment:
I added some info to say that it is implementing an interface, the rest of the things that you are telling me are like this to be coherent with the other strategies that are written similarly.